packages/react-devtools-shared/src/devtools/views/ErrorBoundary/ErrorBoundary.js

```diff
@@ -133,6 +146,35 @@ export default class ErrorBoundary extends Component<Props, State> {
             errorMessage={errorMessage}
           />
         );
+      } else if (isUserError) {
+        return (
+          <CaughtErrorView
+            callStack={callStack}
+            componentStack={componentStack}
+            errorMessage={errorMessage || 'Error occured in inspected element'}
+            info={
+              <>
+                This is likely to be caused by implementation of current
```

Comment:
Nit: could we repeat the `console.error` here to be extra explicit? Ex.

`React DevTools encountered an error while trying to inspect the hooks of this component. This is most likely caused by a developer error in the currently inspected component: ${componentName}

Reply:
I'm not doing this one for now because 
1. user already have that information
<img width="854" alt="image" src="https://user-images.githubusercontent.com/1001890/167018693-d3db4298-ac1d-4b58-9fa9-35014b79d6c9.png">
2. This would require either passing `store` and `selectedElementID` from parent or getting them from context. This component seems problematic because it's currently used as both a general purpose error boundary and an error boundary just for `<InspectedElement>`. I'd like to refactor that but let's do it in a separate PR.

Reply:
Sounds good! What do you think about mentioning the hooks explicitly in the error message still though? 

`React DevTools encountered an error while trying to inspect the hooks of this component. This is most likely caused by a developer error in the currently inspected component`

Regardless if you want to keep your current wording... a couple of grammar nits 😅

`This is likely caused by the implementation of the currently inspected element. Please see your console for the logged error`

Reply:
done!